test_cosi.sh

```diff
@@ -1,10 +1,14 @@
 #!/usr/bin/env bash
 
-# Set to a non-empty value to print debugging messages
-DBG_RUN=1
-. $GOPATH/src/gopkg.in/dedis/cothority.v0/app/libtest.sh
+DBG_SHOW=1
+# Debug-level for app
+DBG_APP=2
+# Uncomment to build in local dir
 STATICDIR=test
 
+. $GOPATH/src/github.com/dedis/cothority/app/lib/test/libtest.sh
```

Comment:
Are we sure the file exists  when running the script ? `dedis/cosi` use `app/client/lib/config` but that's it. Just asking, I did not check in depth...